datasets/th/designated_person/crawler.py

```diff
@@ -77,7 +80,7 @@ def crawl_item(url: str, context: Context):
 
 
 def crawl(context: Context):
-    response = context.fetch_html(context.data_url)
+    response = context.fetch_html(context.data_url, cache_days=2)
```

Comment:
we don't want to wait 2 days to add a newly-sanctioned person, so we don't normally cache these for longer than a day